src/EventStore.Core/EventStore.Core.csproj

```diff
@@ -38,6 +38,7 @@
 		<PackageReference Include="System.Text.RegularExpressions" Version="4.3.1" />
 		<PackageReference Include="System.Threading.Channels" Version="9.0.0" />
 		<PackageReference Include="Scrutor" Version="5.0.2" />
+		<PackageReference Include="AWSSDK.S3" Version="3.7.410.6" />
```

Comment:
```suggestion
```

i think this is already included via the fluent package

Reply:
good spot, fixed :+1: